src/main/webapp/js/userMapData.json

```diff
@@ -250,6 +273,7 @@
 ["Carnegie Mellon University","Qatar"],
 ["Qatar University","Qatar"],
 ["Deltacredit","Russia"],
+["Higher School of Economics","Russia"],
 ["National Research University","Russia"],
 ["NRU-Higher School of Economics","Russia"],
```

Comment:
This looks same as the one we added above. May be remove this one and keep the one we added?